Simplenote/SPAppDelegate.m

```diff
@@ -128,7 +126,10 @@ - (void)setupThemeNotifications
 
 - (BOOL)application:(UIApplication *)application willFinishLaunchingWithOptions:(NSDictionary<UIApplicationLaunchOptionsKey,id> *)launchOptions
 {
-
+    self.coreDataManager = [[CoreDataManager alloc] init];
+    SharedStorageMigrator *storageMigrator = [[SharedStorageMigrator alloc] init];
+    [storageMigrator performMigrationIfNeeded];
```

Comment:
Ideally, IDEALLY, `SharedStorageMigrator` is the very first line we run, and is 100% decoupled from the rest of the app.